performance/config.py

```diff
@@ -8,14 +8,16 @@
 
 from performance.tests.fixtures import get_sample_rp_mapping  # TODO fixture should move in here
 
-# logging.basicConfig(level=logging.DEBUG)  #noqa
+logging.basicConfig(level=logging.INFO)  # noqa
 
 BASE_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
 
 
 class Config:
     ENV = 'prod'
 
+    S3_REPORTS_BUCKET = 'govukverify-hub-prod-billing-reports'
```

Comment:
not blocking, but can we please rename this to `S3_BILLING_REPORTS_BUCKET` so it's clearer what reports we are talking about?